doc/subscribe.rst

```diff
@@ -7,11 +7,12 @@ Subscribe
 Dipy_ is a part of the Nipy_ community and we are happy to share the same
 e-mail list. This makes sense as we can share ideas with a broader
 community and boost collaboration across neuroimagers. If you
-want to post to the list you will need first to subscribe to `nipy
+want to post to the list you will need first to subscribe to the `nipy
 mailing list`_. We suggest to begin the subject of your e-mail with
 ``[Dipy]``. This will help us respond faster to your questions.
 
-Best wishes,
-Dipy developers
+Additional help can be found on the Neurostars_ website. You can 
+sometimes catch us on the `dipy gitter`_ channel.
```

Comment:
Just to add: I don't mind revisiting the "ask us for help anywhere you can
find us" policy at some later point, if we find that it doesn't work well
for us. So far, I think that it is working well. See recent conversations
with Pratik, etc.

On Sat, Nov 21, 2015 at 12:40 PM, Ariel Rokem <arokem@gmail.com> wrote:

> I think that it is more than fine for users to find us on any public forum
> that has history saved in some form. That includes gitter, neurostars, or
> the mailing list. We may direct some questions that call for a broader
> discussion to the mailing list. I don't mind mixing user and developer
> discussions into the same forum. On the contrary - that makes users feel
> like they might also be developers, which is one of the goals of the
> project, as I see it - to empower users to make their tools more useful for
> them
>
> As for "catch us" - I don't think this sounds "too informal". We're not an
> insurance company.
>
>
> On Sat, Nov 21, 2015 at 10:31 AM, Eleftherios Garyfallidis <
> notifications@github.com> wrote:
>
>> In doc/subscribe.rst
>> <https://github.com/nipy/dipy/pull/773#discussion_r45549096>:
>>
>> >  mailing list`_. We suggest to begin the subject of your e-mail with
>> >  ``[Dipy]``. This will help us respond faster to your questions.
>> >
>> > -Best wishes,
>> > -Dipy developers
>> > +Additional help can be found on the Neurostars_ website. You can
>> > +sometimes catch us on the `dipy gitter`_ channel.
>>
>> Thx for correcting the README file. Please don't forget to address the
>> comment above the README. Should we say that gitter is more for developers
>> or we want anyone to be able to drop and ask questions? If we do so people
>> maybe able to have difficulties finding information about previously posted
>> questions.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/nipy/dipy/pull/773/files#r45549096>.
>>
>
>